framework/build.gradle

```
@@ -20,6 +20,13 @@ ext {
     apply from: 'cordova.gradle'
     cdvCompileSdkVersion = privateHelpers.getProjectTarget()
     cdvBuildToolsVersion = privateHelpers.findLatestInstalledBuildTools()
+    if (project.hasProperty('cdvMinSdkVersion')) {
+        cdvMinSdkVersion = Integer.parseInt('' + cdvMinSdkVersion)
+        println "[Cordova] CdvMinSdkVersion is overriden, try it at your own risk."
```

```suggestion
        println "[Cordova] cdvMinSdkVersion is overridden, try it at your own risk."
```

"overridden" typo.

Additionally, I think we should ensure that cdvMinSdkVersion matches case to avoid confusion. These variables can be provided by the user via [these methods](https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html#setting-gradle-properties).

---

Absolutely correct.